doc/syslog/syslog-design.md

```diff
@@ -107,69 +128,97 @@ This might be useful for security reasons.
 
-| Parameter  | Default  | Description                                                                               |
-|:-----------|:---------|:------------------------------------------------------------------------------------------|
-| target     | none     | Name or IP-Address of the system that shall receive messages. Any resolvable name is fine |
-| address    | none     | Bind socket to a given local IP address. This option is only supported for UDP, not TCP   |
-| port       | 514      | Name or numerical value of port to use when connecting to target                          |
-| protocol   | udp      | Type of protocol to use for forwarding (e.g., tcp/udp)                                    |
-| device     | none     | Bind socket to given device (e.g., eth0/vrf0)                                             |
-| ipfreebind | 2        | Manages the IP_FREEBIND option on the UDP socket                                          |
+| Parameter          | Default  | Description                                                                   |
+|:-------------------|:---------|:------------------------------------------------------------------------------|
+| format             | standard | template format                                                               |
+| trap               | notice   | messages with severity equal or grater then this severity will be forwarded   |
+| welf_firewall_name | hostname | firewall name to be used in template, default is system current hostname      |
+
+**MAN page:** [template](
+    https://www.rsyslog.com/doc/v8-stable/configuration/templates.html
+    "Syslog template configuration"
+)
+
+#### 3.2.1.2. Table 3: Syslog Forwarding Output Module
+
+| Parameter     | Default  | Description                                                                               |
+|:--------------|:---------|:------------------------------------------------------------------------------------------|
+| target        | none     | Name or IP-Address of the system that shall receive messages. Any resolvable name is fine |
+| address       | none     | Bind socket to a given local IP address. This option is only supported for UDP, not TCP   |
+| port          | 514      | Name or numerical value of port to use when connecting to target                          |
+| protocol      | udp      | Type of protocol to use for forwarding (e.g., tcp/udp)                                    |
+| device        | none     | Bind socket to given device (e.g., eth0/vrf0)                                             |
+| ipfreebind    | 2        | Manages the IP_FREEBIND option on the UDP socket                                          |
+| filter        | none     | compares the log against the provided regular expression                                  |
+| priority      | notice   | logs with specified priority and higher will be forwarded                                 |
```

Comment:
Will the default values be added in redis during upgrade scenarios (db_migrator script)? If not, we might end up in inconsistency after upgrade?

Reply:
if the keys are empty we will use the global default value in the template